CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ tvm_option(USE_FALLBACK_STL_MAP "Use TVM's POD compatible Map" OFF)
 tvm_option(USE_ETHOSN "Build with Arm Ethos-N" OFF)
 tvm_option(INDEX_DEFAULT_I64 "Defaults the index datatype to int64" ON)
 tvm_option(USE_LIBBACKTRACE "Build libbacktrace to supply linenumbers on stack traces" AUTO)
+tvm_option(BUILD_STATIC_RUNTIME "Build static version of libtvm_runtime" OFF)
```

Comment:
It would be good also to document it in the template https://github.com/apache/tvm/blob/main/cmake/config.cmake with a little note on how to use it and what to expect.

Reply:
Done.